spec/support/pages/admin/split_show_page.rb

```diff
@@ -3,9 +3,10 @@ class AdminSplitShowPage < SitePrism::Page
 
   element :population_count, "tr.population-row span.population"
   element :update_assignments, "tr.population-row a"
+  element :change_weights, "a.change-weights-link"
+  element :decide_split, "a.split-decision"
```

Comment:
Should we just use class selectors here? Do the tests need to care whether the split decision CTA is a button or a link?